lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -866,6 +896,31 @@ def test_pagination(self):
             }
         )
 
+    @ddt.data(
+        (True, "endorsed_comment"),
+        ("true", "endorsed_comment"),
+        ("1", "endorsed_comment"),
+        (False, "non_endorsed_comment"),
+        ("false", "non_endorsed_comment"),
+        ("0", "non_endorsed_comment"),
+    )
```

Comment:
I think that DRF uses the python "true" so if there is any string that is not null, it will be true. Either way, adding that test case to be sure would be good.